template/android/build.gradle

```diff
@@ -7,13 +7,8 @@ buildscript {
         compileSdkVersion = 31
         targetSdkVersion = 31
 
-        if (System.properties['os.arch'] == "aarch64") {
-            // For M1 Users we need to use the NDK 24 which added support for aarch64
-            ndkVersion = "24.0.8215888"
-        } else {
-            // Otherwise we default to the side-by-side NDK version from AGP.
-            ndkVersion = "21.4.7075529"
-        }
+        // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
+        ndkVersion = "23.1.7779620"
```

Comment:
@cortinico: Any reason why we didn't bump to latest r23? https://github.com/android/ndk/wiki/Unsupported-Downloads#r23c

Reply:
Yes because we want to keep the default version provided by AGP